compiler/next/Makefile.help

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+# Copyright 2020-2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
Oops, I guess there were a few that already had the older years.  Looks like this and compiler/next/Makefile.include, though now I think you can just replace the three year range with a two year one

Reply:
Yep saw that one immediately, big oopsies